demos/mandelbrot/mandel.hpp

```diff
@@ -89,14 +89,15 @@ class MandelbrotCalculator {
 
  private:
   void internal_calc() {
+    static constexpr size_t MAX_ITERS = 500;
+    // Anything above this number is assumed divergent. To do less
+    // computation, this is the _square_ of the maximum absolute value
+    // of a non-divergent number
+    static constexpr num_t DIVERGENCE_LIMIT = num_t(256);
+
     m_q.submit([&](sycl::handler& cgh) {
       auto img_acc = m_img.get_access<sycl::access::mode::discard_write>(cgh);
-
-      constexpr size_t MAX_ITERS = 500;
```

Comment:
> Please no binary files in git, source control is for text sources only (i.e. remove the PNG files).

Github supports PNG files just fine. Are you really against having this in the readme?

Reply:
Because of this

```
mandel.hpp(108,32): error GF0156036: variable 'MAX_ITERS' cannot be implicitly captured in a lambda with no capture-default specified
```

Reply:
Oh, I see, it's used in `how_mandel` which has no default capture specified. I don't think that seems right by MSVC but this is an easy fix :shrug: